python/ray/util/client/__init__.py

```diff
@@ -16,7 +16,7 @@
 
 # This version string is incremented to indicate breaking changes in the
 # protocol that require upgrading the client version.
-CURRENT_PROTOCOL_VERSION = "2021-12-07"
+CURRENT_PROTOCOL_VERSION = "2022-02-07"
```

Comment:
Can you bump this to 2022-02-14? 

It looks like it went from 2021-12-07 -> 2022-02-07 in a different PR